src/systems/userinput/paths.js

```diff
@@ -42,6 +45,8 @@ paths.actions.cursor.scalePenTip = "/actions/cursorScalePenTip";
 paths.actions.cursor.scaleGrabbedGrabbable = "/actions/cursorScaleGrabbedGrabbable";
 paths.actions.cursor.mediaVolumeMod = "/actions/cursor/mediaVolumeMod";
 paths.actions.cursor.takeSnapshot = "/actions/cursorTakeSnapshot";
+paths.actions.pen = {};
+paths.actions.pen.remove = "/actions/pen/remove";
```

Comment:
Does this need to be broken out for leftHand, rightHand, and cursor?

Reply:
nope, the removal only happens via the HUD interaction on 2D screens and on 3dof on Go